queries/r/highlights.scm

```diff
@@ -48,10 +48,12 @@
   "&"
   ":"
   "~"
-  "|>"
 ] @operator)
 
-(special) @operator
```

Comment:
Is there any difference between those two groups of operators?

Reply:
`|>` has a different rule than `binary`, the rhs should be a call function.

`(special)` has the same rules than `binary` group. I believe it was put in another group because this kind of operator is not part of the language grammar.